### PR TITLE
Usagov 1248 shell additions

### PR DESCRIPTION
**Pull request by arpage**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1248

## Description

Add nano and tome log list  helper alias.

## Type of Changes

- [ ] New Feature
- [ ] Bugfix
- [ ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [ ] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [x] Infrastructure
  - [x] CMS
  - [ ] WAF
  - [ ] Egress
  - [ ] Tools
- [ ] Other

## Testing Instructions

1. Gain access to the cms terminal:
-  If testing in one of the cloud foundry environments use `cf ssh cms`
- If testing locally build as usual, then use `docker exec -it cms sh`
2. Load the global profile with the command `source /etc/profile`
3. The text editor `nano` should be available.
4. The command `tll` should tail the latest *.log file present in the current directory.  If no log files are present you should see a usage message.

### Requires New Config
- [ ] Yes
- [x] No

### Requires New Content
- [ ] Yes
- [x] No

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
6. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: .docker/Dockerfile-cms [hunk 1]
```diff
@@ -266,7 +267,8 @@ RUN chmod -R +x /etc/periodic \
     && echo "    containertag: " $CONTAINERTAG >> /etc/motd \
     && echo >> /etc/motd \
     && cp /etc/motd /var/www/web/version.txt \
-    && echo "PATH=$PATH" > /etc/profile.d/drupal.sh
+    && echo "PATH=$PATH" > /etc/profile.d/drupal.sh \
+    && echo "alias tll='less $(ls -tr *.log | tail -1)'" > /etc/profile.d/usagov-aliases.sh
```
Comment (akf) [inline .docker/Dockerfile-cms]: Testing this on a local build, I see two problems: 

1. The alias isn't written out properly; I get `alias tll='less '` We need a backslash before the $. 
2. Once I've added the backslash, `tll` still doesn't do what's desired. I think what we really want is `tail -f $(ls -tr *.log | tail -1)`

**Review by akf — CHANGES_REQUESTED** [review]
See my comments on the `tll` alias.

File: .docker/Dockerfile-cms (hunk 1 above)
Comment (arpage) [reply .docker/Dockerfile-cms]: fixed and tested on Mac

**Review by akf — APPROVED** [review]
It works!

**Pull request merged**